### PR TITLE
Desktop Edge 2.2.1.0 release notes

### PR DESCRIPTION
**Pull request by usha-mandya**
### Proposed changes

Release notes for Desktop Edge 2.2.10 release

### Unreleased project version (optional)

Docker Desktop Edge 2.2.1.0

### Related issues (optional)

https://docker.atlassian.net/browse/DESKTOP-2273

### CODE REVIEW

File: docker-for-windows/edge-release-notes.md [hunk 1]
```diff
@@ -12,6 +12,47 @@ For information about Stable releases, see the [Stable release
...
+- Fixed a race condition when starting the WSL engine which caused Docker Desktop to incorrectly report that the containers have exited. Fixes [docker/for-win#5607](https://github.com/docker/for-win/issues/5607).
+- Fixed an issue where editing code inside a container resulted in an error. Fixes [docker/for-win#5528](https://github.com/docker/for-win/issues/5528).
+- Fixed a bug where running the command `DockerCli.exe -SharedDrives` failed to display a list of drives that are shared. Fixes [docker/for-win#5625](https://github.com/docker/for-win/issues/5625).
+
+### Known issues
+
+- DockerNAT has been removed from Docker Desktop 2.2.0.0 as using an IP address to communicate from the host to a container is not a supported feature. To communicate from a container to the host, you must use the special DNS name `host.docker.internal`. We also recommend using ports to communicate from the host to a container. For more information, see [Networking](/docker-for-win/networking/#use-cases-and-workarounds).
+
+  However, if your current setup relies on IP addresses for communication, you can use a temporary workaround to reinstate DockerNAT. To do this, open
```
Comment (nebuk89) [inline docker-for-windows/edge-release-notes.md]: Is this correct for this edge? I thought we had removed this totally for edge now?

Comment (stephen-turner) [reply docker-for-windows/edge-release-notes.md]: This is the edge equal to the stable, not the edge with win 10 home from master branch, so it's correct.

File: docker-for-windows/edge-release-notes.md [hunk 2]
```diff
@@ -12,6 +12,47 @@ For information about Stable releases, see the [Stable release
...
+
+- [Docker Compose 1.25.4](https://github.com/docker/compose/releases/tag/1.25.4)
+- [Go 1.12.16](https://golang.org/doc/devel/release.html#go1.12)
+
+### Bug fixes and minor changes
+
+- Fixed an issue that prevented users from creating files with special characters in the filenames within a shared volume. Fixes [docker/for-win#5520](https://github.com/docker/for-win/issues/5520).
+- Fixed handling of shared volumes with relative paths in `docker-compose.yml`. Fixes [docker/for-win#5516](https://github.com/docker/for-win/issues/5516).
+- Fixed handling of shared volumes where the path case (uppercase / lowercase) did not exactly match the host. Fixes [docker/for-win#5516](https://github.com/docker/for-win/issues/5516).
+- Fixed an issue where changing a file in the Windows file system did not update the file within the container. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530) and [docker/for-win#5550](https://github.com/docker/for-win/issues/5550).
```
Comment (stephen-turner) [inline docker-for-windows/edge-release-notes.md]: Add 5318 as well, which was the earliest reference to it.

Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md]: Fixed.

**Pull request merged**